app/src/test/java/de/westnordost/streetcomplete/quests/opening_hours/AddOpeningHoursTest.kt

```diff
@@ -64,6 +99,15 @@ class AddOpeningHoursTest {
         )
     }
 
+    @Test fun `apply no opening hours sign answer when there was an always open answer before`() {
+        questType.verifyAnswer(
+            mapOf("opening_hours" to "24/7"),
+            NoOpeningHoursSign,
+            StringMapEntryAdd("opening_hours:signed", "no"),
+            StringMapEntryAdd("check_date:opening_hours", LocalDate.now().toCheckDateString())
```

Comment:
Likewise, and more so here.

Reply:
maybe mention it is some comment why SC is doing this? Or point to the comment explaining what is going on?